lib/scss/stanford_events.node.scss

```diff
@@ -25,6 +24,16 @@
         color: $su-color-black;
       }
     }
+
+    .su-event-label-past {
+      color: #7f7776; //$su-color-accent-stone
```

Comment:
shouldn't we use the variable instead of the hex value?

Reply:
Yes we should.  That new color code hasn't been merged.

Reply:
That variable hasn't been merged into Decanter yet.

Reply:
Cool. That's ok then.